internal/schemas.ts

```diff
@@ -678,7 +678,8 @@ const CameraCalibration: FoxgloveMessageSchema = {
     {
       name: "distortion_model",
       type: { type: "primitive", name: "string" },
-      description: "Name of distortion model",
+      description:
+        "Name of distortion model\n\nValues supported by Foxglove Studio: `plumb_bob` and `rational_polynomial`",
```

Comment:
_"Values supported by Foxglove Studio"_ implies to me that these schemas are some general purpose framework that is incidentally used by Foxglove Studio. Imo that is backwards, these schemas are the primary API for rendering things in Foxglove Studio, so it's weird using language as if they are a separate product.

So I would have said something like: 

```suggestion
        "Name of distortion model\n\nSupported values: `plumb_bob`, `rational_polynomial`",
```